Description: Add /usr/lib/xpra to sys.path for xpra script and /usr/lib/parti 
 to sys.path for parti script.
Author: أحمد المحمودي (Ahmed El-Mahmoudy) <aelmahmoudy@sabily.org>
Forwarded: not-needed
--- a/scripts/parti
+++ b/scripts/parti
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 
 import sys
+sys.path.append("/usr/lib/parti")
 import parti.scripts.main
 
 parti.scripts.main.main(sys.argv)
--- a/scripts/parti-repl
+++ b/scripts/parti-repl
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 
 import sys
+sys.path.append("/usr/lib/parti")
 import parti.scripts.repl
 
 parti.scripts.repl.main(sys.argv)
--- a/scripts/xpra
+++ b/scripts/xpra
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 
 import sys
+sys.path.append("/usr/lib/xpra")
 import xpra.scripts.main
 
 xpra.scripts.main.main(__file__, sys.argv)
--- scripts/xpra_launcher
+++ scripts/xpra_launcher
@@ -1,5 +1,7 @@
 #!/usr/bin/env python
 
+import sys
+sys.path.append("/usr/lib/xpra")
 from xpra.scripts.client_launcher import main
 
 main()
